base/strings/util.jl

```diff
@@ -400,6 +400,7 @@ single character, a vector or a set of characters, a string, or a regular expres
 is a function, each occurrence is replaced with `r(s)` where `s` is the matched substring.
 If `pat` is a regular expression and `r` is a `SubstitutionString`, then capture group
 references in `r` are replaced with the corresponding matched text.
+To delete patterns in `pat` set `r` to an empty String (`""` or `string()`).
```

Comment:
Of course, that doesn't make any sense. I should never comment when it's too late in the evening...

Removing `string()` is still a good idea IMHO.